Web/starter_project_vanguard/data/db.json

```diff
@@ -0,0 +1,50 @@
+{
+  "social-projects": [
+    {
+      "image": "projects/hakimhub.svg",
```

Comment:
store only the name of the image file here so that instead of writing projects/ repeatedly you can use "projects/{ name of the image file }' on the components